test/test_schema.ts

```diff
@@ -70,6 +70,27 @@ export interface TestSchema extends Schema {
         };
       };
     };
+    'POST /jsonify': {
+      request: {
+        body: {
+          /**
+           * @type string
+           * @format date-time
+           */
```

Comment:
this is probably not needed, TJS should generate it based on `Date` anyways

Reply:
Yeah, I was just testing/adding support for `ajv-plugins`